scripts/check_deps.sh

```diff
@@ -27,4 +25,5 @@ echo $total_checksum
 # Ignore whitespaces
 if [ "$(echo "$total_checksum" | tr -d '[:space:]')" != "$(echo "$existing_checksum" | tr -d '[:space:]')" ]; then
   echo "ERROR: you need to (run ./pkgs/generate.sh) and commit the changes."
+  echo "$total_checksum" > ./scripts/deps_checksum
```

Comment:
Do we know why our feature branch changes this file? I think we should just "Copy and paste" the revision `develop` is currently using, so the merge is easier.

Reply:
I changed it because the script wasn't detecting what the current state of dependencies are. It does now.

Reply:
Great, in which case let's just move to the revision in `develop`. Less merge pain 😉